documentation/browser-preview.workspace.adoc

```diff
@@ -0,0 +1,94 @@
+
+With that context, read on to learn more about the big new updates in this preview:
+
+- New Cypher editor
+- Richer table visualization
+- More scalable graph visualization
+- New history and quick search
+- Improved saved Cypher experience
+- Revised parameters 
+
+== New Cypher editor
+
+One of the biggest improvements is the new editor, now built from the same source as the underlying Cypher Language. With  Cypher evolving more rapidly than ever before, it’s never been more important to keep pace with the latest language features. Here are a few ways the new editor helps you do that:  
+ 
+Syntax highlighting:: Richer syntax highlighting all but eliminates false positives from things like property values being misinterpreted as Cypher keywords and differentiates more tokens for ease of reading.
+
+image::cypher-editor-syntax-highlighting.png[]
+
+
+Auto completions:: The underpinnings of the Cypher editor now have a much improved understanding of valid constructs, meaning it will, for example, far more accurately complete the next valid keyword.
+
+image::cypher-editor-auto-completions.gif[]
+
+Inline help:: When using a procedure or making a function call, helpful descriptions and signatures are presented, allowing you to better understand what’s expected as an input or what might be returned.
+
+image::cypher-editor-inline-help.png[]
+
+Cypher linting:: One of the single most useful new features we’ve added to the editor is Cypher linting, so you’ll now see useful warnings and errors _before_ running a query. This really helps shorten the feedback loop, meaning you don’t have to run a query to spot syntactic or semantic errors. This is especially useful when editing larger queries or using new language constructs where you might not be so familiar with what to expect.
+
+image::cypher-editor-linting.png[]
+
+Cypher reference:: As a complement to the in-editor help, we now also provide a searchable Cypher reference in the sidebar. This is useful if you want to check the syntax of a Cypher construct or want a refresher, for example, of which temporal functions exist and how to use them. The reference also provides links to the cypher manual for more in depth explanations and examples.
+
+image::cypher-reference.gif[]
+
+== Richer table visualization
+
+The table visualization now employs reactive result fetching by default, meaning we’ll automatically limit the number of rows returned by a query regardless of any `LIMIT` statements you apply. This is useful for new and experienced users alike, where an unintentional omission of a `LIMIT` statement could result in the results returned could previously overwhelm Browser. The table now also employs virtualized rendering meaning that even if you do return a larger number of rows, we only render those in view to keep the application more responsive.
```

Comment:
```suggestion
The table visualization now employs reactive result fetching by default, meaning that the number of rows returned by a query is automatically limited, regardless of any `LIMIT` statements you apply. 
This is useful for both new and experienced users since an unintentional omission of a `LIMIT` statement can lead to an unexpectedly large amount of results returned, which could previously have overwhelmed Browser.
The table now also employs virtualized rendering meaning that even if you do return a larger number of rows, only those in view are rendered to keep the application more responsive.
```